geometry_msgs/PoseArray GoalArray
---
float32[] probs